core/controllers/admin.py

```diff
@@ -266,7 +266,7 @@ def _generate_dummy_explorations(
             exploration_ids_to_publish = []
             for i in range(num_dummy_exps_to_generate):
                 title = random.choice(possible_titles)
-                category = random.choice(feconf.SEARCH_DROPDOWN_CATEGORIES)
+                category = random.choice(constants.SEARCH_DROPDOWN_CATEGORIES)
```

Comment:
@seanlip, are we expecting these changes to be reviewed by you?

Reply:
That's a good question. I would have expected that to be the case, yeah. Any idea why CODEOWNERS is not picking that up?

Reply:
@seanlip, now you are there (I don't know how!) but I cannot find any comment in this thread like:
`abhaygupta97 abhaygupta97 requested review from seanlip as code owner`.

Reply:
Weird. Probably github updates these checks lazily, or something :P